packages/Card/src/components/Avatar/Avatar.styles.js

```diff
@@ -15,7 +15,7 @@ export const avatarSizeStyles = {
   `,
 
   medium: css`
-    border-radius: 12px;
+    border-radius: ${tokens.card.borderRadius};
```

Comment:
I discussed with Allison, and have reconsidered my advice for you for this PR.  Let's kill this version of the `<Avatar>` completely.  It means you'll need to update the `changeset` to include this `major` (breaking) change, but the risk is low, and there's a clear migration path – use the `@paprika/avatar` instead.